test/client/js/libs/handlebars/localetimeTest.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const Handlebars = global.Handlebars = require("handlebars");
+const expect = require("chai").expect;
+
+require("../../../../../client/js/libs/handlebars/localetime");
```

Comment:
This is a bit gross but I would like the test hierarchy to mirror the code hierarchy. We can very probably clean that up a bit later, when structure for client-side stuff gets a bit more stable. In the meantime, it's a test, and it passes...